styles/utils.css

```diff
@@ -0,0 +1,105 @@
+/* own utilities */
+
+.rounded-buttons {
+    display: inline-block;
+}
+
+.rounded-button {
+    background: rgb(187, 225, 215);
```

Comment:
use a variable and make the colors consistent (use hex for all of them)

Reply:
Done